packages/js/src/utils/callback-uri.ts

```diff
@@ -23,19 +17,20 @@ export const verifyAndParseCodeFromCallbackUri = (
   if (!callbackUri.startsWith(redirectUri)) {
     throw new LogtoError('callback_uri_verification.redirect_uri_mismatched');
   }
+  const uriParameters = parseUriParameters(callbackUri);
 
-  const authenticationResult = parseCallbackUri(callbackUri);
-  const {
-    code,
-    state: stateFromCallbackUri,
-    error,
-    error_description: errorDescription,
-  } = authenticationResult;
+  const error = conditional(uriParameters.get(QueryKey.Error));
+  const errorDescription = conditional(uriParameters.get(QueryKey.ErrorDescription));
```

Comment:
why do we need `conditional` here?

Reply:
My bad. There is no need to do it now.

(Did it for some nit reason that is invalid now.)

Reply:
Fixed. PTAL @gao-sun .